### ISSUE TITLE
Standardize spacing in challenge files

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/freeCodeCamp#47533]
### Describe the Issue

In the challenge files, some of the whitespace characters are tabs, others are spaces. It looks fine when you open a challenge:
![Screen Shot 2022-09-16 at 9 43 18 AM](https://user-images.githubusercontent.com/20648924/190666213-ee4ef428-5416-49b3-9f22-5431f7a7c27f.png)

But when you create a forum post:
![Screen Shot 2022-09-16 at 9 42 39 AM](https://user-images.githubusercontent.com/20648924/190666380-3562a34d-1374-4a0c-8506-e4c8f333baaf.png)

The code isn't aligned. We should replace all the tabs with two spaces. This is likely present in many challenge files, not sure how many. One example is below.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-25

### Expected behavior

Forum post code to be aligned.

**Comment by ojeytonwilliams** [comment]
I think we should go the other way and standardize to tabs. @bbsmooth pointed out to me that this is better from an a11y perspective, because then it's up to the viewer how much they're indented.

**Comment by ojeytonwilliams** [comment]
Good point. It may not make much difference in this case, because the tab spacing in the editor isn't configurable.

**Comment by ShaunSHamilton** [comment]
> It might be a little trickier to regex replace those, as well.

We should not RegEx anything here. We should update the linter to fix this for us. I know we have some linting rules for Markdown. So, I am sure we can extend them to be specific about code blocks within Markdown.

**Comment by moT01** [comment]
If we can define lint rules - I suppose that would be the way to go. Then a tab at the start shouldn't be too difficult. I still prefer the spaces.

**Comment by ojeytonwilliams** [comment]
If it's customizable, we should be able to use this https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.ICodeEditor.html#onDidChangeConfiguration to persist a setting. So, that should work.

**Comment by naomi-lgbt** [comment]
> I know we have some linting rules for Markdown. So, I am sure we can extend them to be specific about code blocks within Markdown.

We seem to have the rule explicitly turned off:

https://github.com/freeCodeCamp/freeCodeCamp/blob/d6164c6313e51d4537a40e6f73afa48b9969e136/curriculum/challenges/.markdownlint.yaml#L11

**Comment by ojeytonwilliams** [comment]
@naomi-lgbt I don't believe that covers spaces inside code blocks.

**Comment by moT01** [comment]
Some of this conversation feels a little off issue. I found a few files that had some tabs instead of spaces that didn't look good when creating a forum post - seems like it should be a simple fix. It feels a little bit like we are trying to put out a candle with a fire extinguisher. I say we just merge the PR I made to make the files consistent, and we can continue the conversation on some of these other things if we want.
